Source/WebCore/loader/FrameLoader.cpp

```diff
@@ -3187,7 +3187,17 @@ String FrameLoader::userAgent(const URL& url) const
 
     if (userAgent.isEmpty() || m_client->hasCustomUserAgent())
         userAgent = m_client->userAgent(url);
-    
+
+    // FIXME: Move this to a proper UA override singular mechanism
+    // https://bugs.webkit.org/show_bug.cgi?id=274374
```

Comment:
I'd move this comment down to the Firefox string below, and use this location for our normal FIXME about removing the Quirk once we get Easyjet (or is it Bing?) to fix their site.